gui/wxpython/lmgr/layertree.py

```diff
@@ -246,6 +246,14 @@ def _getSelectedLayer(self):
     # for compatibility
     layer_selected = property(fget=_getSelectedLayer)
 
+    def _recalcLyrSettingsBtnPos(self):
```

Comment:
I assume you created this wrapper to put the documentation in there, but at least it should have a more readable name, maybe just '_recalculateButtonPosition' or something like that.

Reply:
All right, I changed it.